src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -124,17 +126,22 @@ class ReportActionContextMenu extends React.Component {
                     );
                 },
             },
-
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: false,
-                onPress: () => {},
+                shouldShow: this.canEdit(),
```

Comment:
Then for this one we can just pass a reference to the function itself:

```suggestion
                shouldShow: this.canEdit,
```

Reply:
whenever I do this change the screen goes all white, I'm not entirely sure on why though

Reply:
Do you need to put something like this in the constructor?

```javascript
this.canEdit = this.canEdit.bind(this);
```

Reply:
yep tht was it, had to move the bindings to the beginning of the constructor as well